ably/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "TBC",
```

Comment:
Please provide the app_uuid

Reply:
Added in e76fcd0368691bd9fbd553a401d1b04ed07a409b